unittests/interface/interface_functions/test_host_bIt_field_router_compressor.py

```diff
@@ -26,7 +26,7 @@ def setUp(self):
 
```

Comment:
Just noticed the name of this file has a capital I in the middle of it rather than lower-case, guess this could be changed here?

Reply:
Fixed